docs/src/filesystem-sysroot.md

```diff
@@ -0,0 +1,69 @@
+# Filesystem: Physical /sysroot
+
+The bootc project uses [ostree](https://github.com/ostreedev/ostree/) as a backend,
+and maps fetched container images to a "deployment": <https://ostreedev.github.io/ostree/deployment/>.
```

Comment:
`[deployment](https://ostreedev.github.io/ostree/deployment)` instead to make a pretty link?